test_upper_case_names (__main__.TestExercice) ... ERROR

======================================================================
ERROR: test_upper_case_names (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 17, in test_upper_case_names
    output = list(map(exercice.majuscule, self.words))
  File "/home/runner/work/c02-ch2-majuscule-Depdx/c02-ch2-majuscule-Depdx/exercice.py", line 5, in majuscule
    delta_maj_min = ord(a)-ord(A)
NameError: name 'a' is not defined

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
